### PR TITLE
Add code of conduct

### LINKED ISSUE #385: Add a code of conduct to the GWB
Since the GWB has been attracting new users, and is expected to grow more, I think it a good time to add a code of conduct repository. 

The code of conduct of ASPECT is based on https://www.contributor-covenant.org, which looks good to me. I will have to take a closer look at what the differences are between version 1.4 and 2.0 of the code. 

I can't use the exact same code of conduct, since it contains specific CIG language and the GWB is not currently part of CIG. That part can be revised if that happens in the future.

### PR DESCRIPTION
**Pull request by gassmoeller**
This is ASPECT's code of conduct, with a small modification to link to the WB Authors file.

Closes #385.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+individuals that jeopardizes the harmony of the project will need to be
+addressed.
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting any [maintainer](AUTHORS) of the project, or CIG leadership at <private@geodynamics.org>. CIG leadership will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. CIG leadership is obligated to maintain confidentiality with regard to the reporter of an incident, although confidentiality cannot be promised in all situations under the Title IX law of the United States. Further details of specific enforcement policies may be posted separately.
```
Comment (MFraters) [inline CODE_OF_CONDUCT.md:47]: @gassmoeller @ljhwang What do we do with the private@geodynamics.org?

**Review by MFraters — APPROVED** [review]
Thanks for making this @gassmoeller! We should figure out what to do with the private@geodynamics.org issue before merging, but otherwise looks good to me. @ljhwang, can you also have a look at this?

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (ljhwang) [reply CODE_OF_CONDUCT.md:47]: I have requested that the email address be set-up.  Thanks @gassmoeller for adding this.

**Comment by MFraters** [comment]
Great, thanks we can merge this then.

**Pull request merged**